modules/monitor/apm/topology/topology.go

```diff
@@ -1358,6 +1356,7 @@ func (topology *provider) Services(serviceName string, nodes []*Node) []ServiceD
 
 type ServiceInstance struct {
 	ApplicationName     string `json:"applicationName,omitempty"`
+	ServiceId           string `json:"serviceId,omitempty"`
```

Comment:
Proper nouns need to be capitalized, e.g. ID

Reply:
Combining id with other fields is universal, and id as a separate field needs to be all uppercase

Reply:
> Proper nouns need to be capitalized, e.g. ID
Combining id with other fields is universal, and id as a separate field needs to be all uppercase

Reply:
> Proper nouns need to be capitalized, e.g. ID

Combining id with other fields is universal, and id as a separate field needs to be all uppercase